src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -378,6 +395,112 @@
                 return this;
             },
 
+            loadToolbar: function(token, newToken) {
+                newToken = (newToken || token);
+                this.load(
+                    'sfwdt' + token,
+                    '{{ path("_wdt", { "token": "xxxxxx" }) }}'.replace(/xxxxxx/, newToken),
```

Comment:
no need for .replace() if newToken is not set, you can generate right URL from token directly.

Reply:
missd the newToken assignment above.. but still not really needed without newToken.